### ISSUE TITLE
how to work with php sdk v3

### ISSUE THREAD

**Issue by ddancy** [ienzam/s3-multipart-upload-browser#9]
Hi, thank you for your contribution. Your code only support for v2.4.0. But now, it seems we can't download this version from aws. Now current version is v3. In this version, the $command->prepare() can't be used. I tried lots to make the code run successfully, but failed. Do you know how to work with v3 by using your code?

**Comment by mattwa1sh** [comment]
Yeah I'm having the same issue. There doesn't seem to be much out there about what seems to me a common use case. Any help would be appreciated.

**Comment by ienzam** [comment]
Sorry, this project is not actively developed anymore.

**Comment by abiusx** [comment]
This new project takes care of the problem:
https://github.com/zdresearch/s3-multipart-upload-javascript-browser
